include/zephyr/bluetooth/hci.h

```diff
@@ -97,6 +97,15 @@ int bt_hci_get_conn_handle(const struct bt_conn *conn, uint16_t *conn_handle);
  */
 int bt_hci_get_adv_handle(const struct bt_le_ext_adv *adv, uint8_t *adv_handle);
 
+/** @brief Get periodic advertising sync handle for an advertising set.
```

Comment:
"advertising set" feels like the wrong description here, but I dont know if there is anything better.

Reply:
I updated the name to `Get periodic advertising sync handle`, I think it's better like that.